### ISSUE TITLE
Pytorch Dataset Label Integration

### ISSUE THREAD

**Issue by Oufattole** [Oufattole/meds-torch#2]
https://github.com/Oufattole/meds-torch/blob/d19ae01e2393ec0281919b70fb7a160f9c36e340/src/MEDS_torch/pytorch_dataset.py#L275

We need to add support for supervised training using labels to generate windows of input data for the model.
